obal/data/roles/setup_sources/tasks/git.yml

```diff
@@ -14,4 +14,4 @@
 - name: 'Register source dir tito argument'
   set_fact:
     build_package_tito_releaser_args:
-    - '--arg source_dir={{ setup_sources_git_checkout_dir }}'
+    - 'source_dir={{ setup_sources_git_checkout_dir }}'
```

Comment:
I had some doubts about changing this part. Should we merge a matching change in foreman-packaging at the same time or write compatibility code?